docs/about.rst

```diff
@@ -3,4 +3,16 @@ Why `repytah`
 
 Music-based data streams often have repeated elements that build on each other, creating hierarchies. 
 Therefore, the goal of the Python package `repytah` is to extract these repetitions and their relationships 
-to each other in order to form aligned hierarchies.
\ No newline at end of file
+to each other in order to form aligned hierarchies, a low-dimensional representation for sequential data
+that synthesizes and aligns all possible hierarchies of the meaningful repetitive structure in a song along
+a common time axis. In addition to visualizing the structure of a song, aligned hierarchies can be
+embedded into a classification space with a natural notion of distance and post-processed to narrow the
+exploration of a music-based data stream to certain lengths of structure, or to address numerous MIR
+tasks, including the cover song task, the segmentation task, and the chorus detection task.
+

```

Comment:
Should we mention that our package can go beyond the music area after the first paragraph? Something like this: Moreover, the python package `repytah` can go beyond the MIR area. People working with sequential data where repetitions have meaning can use `repytah` as a tool as well.

Reply:
Good idea!